### PR TITLE
Fix missing body

### PR DESCRIPTION
**Pull request by georgeismike**

### CODE REVIEW

File: spec/conduit/telispire/actions/query_device_info_spec.rb [hunk 1]
```diff
@@ -56,6 +56,7 @@
 
     subject                 { query_device_info.perform }
     it                      { should be_an_instance_of QueryDeviceInfo::Parser }
+    its(:body)              {should_not eq ""}
```
Comment (captainbeardo) [inline spec/conduit/telispire/actions/query_device_info_spec.rb]: Should have spaces on the inside of the braces. Maybe test it with something other than empty too?

Comment (georgeismike) [inline spec/conduit/telispire/actions/query_device_info_spec.rb]: @captainbeardo this is the issue.. https://app.honeybadger.io/projects/44725/faults/31646306#notice-summary

Which to me begs the question that our conduit tests or inheritance is not ensuring that we have expected interfaces like `response.body`.  Seems like we need to build something that says. "this is a conduit carrier driver and should have these interfaces with tests" right?

Comment (captainbeardo) [inline spec/conduit/telispire/actions/query_device_info_spec.rb]: I agree with that assessment. It could probably be done with a shared example set.

Comment (captainbeardo) [inline spec/conduit/telispire/actions/query_device_info_spec.rb]: I agree, it seems like something we could do with a shared rspec example.

**Pull request merged**